lib/ansible/vars/__init__.py

```diff
@@ -150,6 +153,9 @@ def extra_vars(self, value):
     def set_inventory(self, inventory):
         self._inventory = inventory
 
+    def set_check(self, check):
```

Comment:
I think we should have a more 'generic' option and allow several vars to be set from options, not just this one